packages/mobile_broadband_provider_info.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Mobile_broadband_provider_info < Package
+class Mobile_broadband_provider_info < Autotools
   description 'Network Management daemon'
   homepage 'https://gitlab.gnome.org/GNOME/mobile-broadband-provider-info'
   version '20201225'
```

Comment:
Update to [20230416](https://gitlab.gnome.org/GNOME/mobile-broadband-provider-info/-/tags/20230416)?